build.sbt

```diff
@@ -339,3 +339,40 @@ lazy val unidocSettings = Seq(
     }
   },
 )
+
+lazy val genExamplesSettings = Seq(
+  Compile / genExamples := {
+    val cp = (runtime2 / Test / dependencyClasspath).value
+    val inSrc = (runtime2 / Compile / sources).value
+    val inRSrc = (runtime2 / Test / resources).value
+    val stream = (runtime2 / streams).value
+    val filesToWatch = (inSrc ++ inRSrc).toSet
```

Comment:
Does this need to watch more than just the sources and resources? For example, what if a scala file in daffodil-core changes? Couldn't that potentially require the need for a rebuild? Seems any changes to anything on the classpath would need to be watched?

Reply:
Fair enough, I guess the extra complexity isn't worth it with the unlikeliness of actualy needing a rebuild, and the likelyhood of catching it in the github actions if it was missed. Fine with me.